mem-03-D.xml

```diff
@@ -11625,7 +11625,7 @@ The logo of Star Alliance, the largest airline alliance on Earth, is often paint
       <column name="definition_fi">keskipäivä</column>
       <column name="synonyms">{pemjep:n}</column>
       <column name="antonyms">{ramjep:n:1}</column>
-      <column name="see_also">{megh:n}, {po:n}, {pov:n}</column>
+      <column name="see_also">{megh:n}, {po:n:1}, {pov:n}</column>
```

Comment:
This should be `{po:n:1h}` to "hide" the number in the link. This is down (maybe not completely consistently) for homophones which clash with names. For example, see `paq:n`.

Actually, I've updated the links to `po:n` already, so if you merge with master, it should be fixed.

Reply:
Yeah, this series already includes a merge commit: https://github.com/De7vID/klingon-assistant-data/pull/762/commits/93565bcda68ac9e8ffc4b95505d000e05f022cdc

I'll just rebase it and incorporate the tag fix from your other comment.